Cargo.toml

```diff
@@ -41,6 +41,7 @@ version = "3.0.0"
 hex = "0.4.2"
 picky = "5.0.0"
 psa-crypto = { version = "0.2.1" , default-features = false, features = ["with-mbed-crypto"], optional = true }
+zeroize = { version = "1.1.0", features = ["zeroize_derive"] }
```

Comment:
Should this be changed to a public export in `parsec-interface`, to match `secrecy`?

Reply:
No, not really - the reasoning for only exporting `secrecy` despite using types from both in the operations is that for `Secret` you *need* to have access to `secrecy` in order to create or use an object, you can't just do `.into()` from the right type (like for `Zeroizing`). Also, if you want to access the data in `Secret` you need the `ExposeSecret` trait, whereas `Zeroizing` has `AsRef` and `AsMut` implemented so it can be used easily.

There is a point to be made that someone seeing `Zeroizing` might try and do `Zeroizing::new(...)` - don't know how we could discourage that. Maybe in the interface documentation.